src/session/content/message_row/indicators_model.rs

```diff
@@ -0,0 +1,114 @@
+use gettextrs::gettext;
```

Comment:
Am I correct, that this file is related to #316?

Reply:
Yep. It's an easier way to write the "glue code" for the widget. In this particular case it's not that useful, but it will when we start implementing more things like the "edited" text, the "single" reaction in private chats, message status, view count, etc.